.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+    types: [ opened, reopened, synchronize, labeled ]
```

Comment:
I would just remove `types` from pull request, have it on everything till we don't need something